### ISSUE TITLE
VSC - PHP CS Fixer: php general error.

### ISSUE THREAD

**Issue by anyav77** [susanBuck/e2-fall19#102]
When saving vhosts file, I got this error:

> PHP CS Fixer: php general error.

Disabling/enabling the extension didn't help.

Prior to that, I added blog directory to E2 workspace

![php cs fixer error](https://user-images.githubusercontent.com/1045307/70398498-f014ac00-19d0-11ea-8f3a-a76ecaf1b6b7.png)

**Comment by susanBuck** [comment]
When this happens, try going to _Help > Toggle Developer Tools_.

It'll bring up a developer tools window, just like you'd see in Chrome. 

Go to the **Console** tab. Here you should see more details about the error.

**Comment by susanBuck** [comment]
I just got the same popup in VSCode.

I opened _Developer Tools_ and found the error:

![image](https://user-images.githubusercontent.com/237711/70548129-0a20cc80-1b40-11ea-8c1c-db642079a3bb.png)

The error notes it can't find `/Users/Susan/.vscode/extensions/junstyle.php-cs-fixer-0.1.127/php-cs-fixer.phar`, so I navigated to `/Users/Susan/.vscode/extensions/` to investigate. I found that the php-cs-fixer plugin had updated to `0.1.129`.

![image](https://user-images.githubusercontent.com/237711/70548328-58ce6680-1b40-11ea-9314-17756bd9b764.png)

To fix, I opened my VSCode `settings.json` file and corrected the path:

![image](https://user-images.githubusercontent.com/237711/70548236-33d9f380-1b40-11ea-9045-2071547fa534.png)

Can't say for sure, but I suspect the above is the same cause of your error. : )
